src/interfaces/IICS20Transfer.sol

```diff
@@ -7,22 +7,21 @@ import { IICS20TransferMsgs } from "../msgs/IICS20TransferMsgs.sol";
 interface IICS20Transfer is IICS20TransferMsgs {
     /// @notice Called when a packet is handled in onSendPacket and a transfer has been initiated
     /// @param packetData The transfer packet data
-    event ICS20Transfer(ICS20Lib.UnwrappedFungibleTokenPacketData packetData);
+    event ICS20Transfer(ICS20Lib.SendPacketData packetData);
```

Comment:
That's a general comment. As we have specific files for errors, I have seen many projects having the specific files for events too. We should consider it.